MdeModulePkg/Include/Guid/ArmFfaRxTxBufferInfo.h

```diff
@@ -14,16 +14,22 @@
  */
 typedef struct ArmFfaRxTxBuffersInfo {
   /// Tx Buffer Address.
-  VOID      *TxBufferAddr;
+  VOID       *TxBufferAddr;

```

Comment:
Since we are touching these fields, I think it would make more sense to change them over to EFI_PHYSICAL_ADDRESS. It will save a lot hassle below that involves double pointers.